<?xml version="1.0" ?>
<launch>
  <arg name="robot_ip" default="robot.franka.de" />
  <arg name="arm_id" value="franka_emika" />
  <include file="$(find franka_hw)/launch/franka_hw.launch" >
    <arg name="robot_ip" value="$(arg robot_ip)" />
    <arg name="arm_id" value="$(arg arm_id)" />
  </include>
  <group ns="$(arg arm_id)">
      <rosparam command="load" file="$(find franka_example_controllers)/config/franka_example_controllers.yaml" />
      <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"  args="model_example_controller --shutdown-timeout 2"/>
      <node pkg="rviz" type="rviz" output="screen" name="rviz" args="-d $(find franka_description)/rviz/franka_description.rviz"/>
    </group>
</launch>
